### ISSUE TITLE
Error Restarting Apache on Production Server

### ISSUE THREAD

**Issue by dnoren32** [susanBuck/dwa15-fall2018#68]
As I follow the steps for getting everything set up this week, I hit a road bump when I try to restart Apache.

`root@lamp-s-1vcpu-1gb-nyc1-01:/var/www/html/foobooks# service apache2 restart
Job for apache2.service failed because the control process exited with error code.See "systemctl status apache2.service" and "journalctl -xe" for details.`

I see it's referencing some file but I'm not sure how to get to it. I try accessing the apache error log and get a permission denied message. Any thoughts?

**Comment by susanBuck** [comment]
Right after you SSH into your server it shows a bunch of info. The first line is details about your server software. For example, thus is what mine shows:

```
/Applications/MAMP/htdocs   $ ssh root@159.65.236.201
Welcome to Ubuntu 16.04.3 LTS (GNU/Linux 4.4.0-109-generic x86_64)

[...etc...]
```

Can you share what info you see for this?

Also, I searched the error message you received and found this thread: [Apache not able to restart](https://askubuntu.com/questions/629995/apache-not-able-to-restart).

Many of the answers suggest its b/c of an error in a config file.

Try the suggestions in [this answer first](https://askubuntu.com/a/805767).

**Comment by dnoren32** [comment]
Ok so when I ssh'd in. This is the first line that came up: 
`Welcome to Ubuntu 18.04.1 LTS (GNU/Linux 4.15.0-34-generic x86_64)`

Here is the error I got when I ran `configtest` this what I got: 
`AH00526: Syntax error on line 31 of /etc/apache2/sites-enabled/000-default.conf:
<Directory> directive missing closing '>'
Action 'configtest' failed.
The Apache error log may have more information.'

I found the error and now it works! Thank you!!!!
